packages/web/src/pages/settings-page/components/desktop/ManagerMode/AccountListItem.tsx

```diff
@@ -64,17 +74,17 @@ export const AccountListItem = () => {
   )
 
   const composeChat = useComposeChat({
-    user: user!
+    // @ts-expect-error - This wants a User, but works with UserMetadata
```

Comment:
Can we just update the type in `useComposeChat`?

Reply:
Unfortunately not without a lot of difficulty/a large diff because `useComposeChat` works with a lot of preexisting things that expect a `User` but also could work with just a `UserMetadata`